conf/fedora2-native.xml

```diff
@@ -66,13 +70,9 @@
     <bean id="namespacePrefixMapper" class="org.fcrepo.migration.foxml.NamespacePrefixMapper">
         <constructor-arg name="namespaceFile" type="java.io.File" ref="namespaceFile"/>
     </bean>
-
-    <bean id="fedora4Client" class="org.fcrepo.migration.f4clients.DefaultFedora4Client">
-        <constructor-arg name="repo" ref="repo" />
-    </bean>
-
-    <bean id="repo" class="org.fcrepo.client.impl.FedoraRepositoryImpl">
-        <constructor-arg name="repositoryURL" ref="fedora4Url" />
+    
+    <bean id="fedora4Client" class="org.fcrepo.migration.f4clients.StatelessFedora4Client">
+        <constructor-arg name="fcrepoBaseURL" ref="fedora4Url" />
```

Comment:
I'll keep that in mind, but I'd rather not change the multiple example files, which would in turn invalidate the slides and tutorials we use this content for.  Also, I think extensive style changes within an unrelated ticket makes it harder to make sense of historic commits.  That said, if you want these changes before merging this commit, I'd grudgingly make them so long as *that* commit wasn't squashed.